### PR TITLE
Dev repo cleanup

### PR DESCRIPTION
**Pull request by apyskir**
Remove unused code

### CODE REVIEW

File: src/utils.py [hunk 1]
```diff
@@ -365,36 +310,6 @@ def coco_evaluation(gt_filepath, prediction_filepath, image_ids, category_ids, s
     return cocoEval.stats[0], cocoEval.stats[3]
 
 
-def label(mask):
```
Comment (taraspiotr) [inline src/utils.py]: @apyskir at the moment label_multiclass_image() uses this function.

Comment (apyskir) [reply src/utils.py]: @taraspiotr Yes, but it was a duplicate

**Comment by jakubczakon** [comment]
@apyskir could you please resolve `utils.py` conflicts ?

**Comment by apyskir** [comment]
@jakubczakon Done

**Pull request merged**